lib/jwtoken.rb

```diff
@@ -0,0 +1,55 @@
+module API
+  class JWToken
+    attr_reader :scope_type
+
+    # Initialize constructor
+    #
+    # @returns nothing
+    def initialize(scope_type)
+      @scope_type = scope_type
+    end
+
+
+    # Create JWT base on payload and secret_key
+    #
+    # @returns String
+    def token
+      pl = if scope_type == 'special'
+             payload.merge!(scopes: special_scopes)
+           else scope_type =='general'
```

Comment:
INFO: Style/SpaceAroundOperators: Surrounding space missing for operator `==`. (https://github.com/bbatsov/ruby-style-guide#spaces-operators)